internal/component/loki/source/gcplog/internal/gcplogtarget/push_translation.go

```diff
@@ -48,14 +52,28 @@ func (pm PushMessage) Validate() error {
 	return nil
 }
 
+func (pm PushMessageBody) ID() string {
+	if pm.Message.MessageID != "" {
+		return pm.Message.MessageID
+	}
+	return pm.Message.DeprecatedMessageID
+}
+
+func (pm PushMessageBody) PT() string {
```

Comment:
```suggestion
func (pm PushMessageBody) PublishTimestamp() string {
```

nit: I would have to go to the implementation to know that PT in this case is `PublishTimestamp`

Reply:
Yeah that is true. But I will remove PublishTime for now anyway so I wont need to have this one